pkg/gofr/migration/interface.go

```diff
@@ -0,0 +1,33 @@
+package migration
+
+import (
+	"context"
+	"database/sql"
+	"time"
+
+	goRedis "github.com/redis/go-redis/v9"
+)
+
+type Decorator interface {
```

Comment:
This could be named a bit meaningful!

Reply:
Have removed the interface! as it was not being used externally!